### PR TITLE
docs: updated readme and fixed gopages link

### PR DESCRIPTION
**Pull request by gary1998**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # IBM Cloud Security & Compliance Center Go SDK Version v5.0.0
 
 Go client library to interact with various
-[IBM Cloud Security & Compliance Center APIs](https://cloud.ibm.com/docs?tab=api-docs&category=platform_services%2Csecurity).
+[IBM Cloud Security & Compliance Center APIs](https://cloud.ibm.com/apidocs/security-compliance).
```
Comment (ashpc) [inline README.md:13]: Can't use `&` change IBM Cloud Security and Compliance Center APIs

File: README.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ Go client library to interact with various
 
 <!-- toc -->
 
-- [IBM Cloud Security & Compliance Center Go SDK Version v5.0.0](#ibm-cloud-security--compliance-center-go-sdk-version-v120)
-  - [Table of Contents](#table-of-contents)
+- [IBM Cloud Security & Compliance Center Go SDK Version v5.0.0](#ibm-cloud-security--compliance-center-go-sdk-version-v500)
```
Comment (ashpc) [inline README.md:28]: Same here

File: README.md [hunk 3]
```diff
@@ -43,16 +42,7 @@ Go client library to interact with various
...
-
-| Service Name                                                                         | Package name              |
-| ------------------------------------------------------------------------------------ | ------------------------- |
-| [Configuration Governance](https://cloud.ibm.com/apidocs/security-compliance/config) | configurationgovernancev1 |
-| [Admin Service](https://cloud.ibm.com/apidocs/security-compliance/admin)             | adminserviceapiv1         |
-| [Posture Management](https://cloud.ibm.com/apidocs/security-compliance/posture-v1.0)      | posturemanagementv1       |
-| [Posture Management v2](https://cloud.ibm.com/apidocs/security-compliance/posture-v2.0)      | posturemanagementv2       |
-
-
+The IBM Cloud Security & Compliance Center Go SDK allows developers to programmatically interact with the following IBM Cloud Security and Compliance Center.
```
Comment (ashpc) [inline README.md:45]: Same here

**Review by ashpc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gary1998) [reply README.md]: fixed

**Review by archanaponnada — APPROVED** [review]
changes look fine

**Pull request merged**